docs/wiki/research/PBS/ET.md

```diff
@@ -0,0 +1,138 @@
+Barnabe argues [[3]](#resources) that ePBS in its current proposed version does not answer some fundamental questions:
+- is it the market structure *or* the allocation mechanism that we consider too centralized under MEV-boost?
+- if the answer is the former, is it worth to enshrine the same mechanism that we consider both a technical debt (a bug) and a flawed design philosophy, that only exists now outside of protocol?
+ - wouldn't it be better **for the protocol** to not be split between two worlds and just be concerned about allocating *proposing rights* and not worrying about allocating *building rights*?
+
+![](/docs/wiki/research/img/ET_2worlds.webp)
+
+This is the gist of execution tickets (ETs):
+
+A permissionless market allowing buyers to purchase the right to propose execution payloads. 
+These rights confer the ticket holder (that can be a different party than the Validator) with a random allocation, to propose an execution payload.
+
+## Relationship to PBS/ePBS
+Execution tickets are orthogonal to ePBS, in the sense that a new paradigm is being proposed: 
+a validator–proposer separation, where the proposer is recruited by the protocol and could once again delegate the role of construction to another entity, the builder.
+The delivery of the beacon block, however, would remain the validator's prerogative.
+
+![](/docs/wiki/research/img/ETvsPBS.webp)
+
+One generalization of PBS is proposed by PEPC [[1]](#resources) [[8]](#resources) and has generated versions of ePBS, like slot-auction ePBS, which added a subtle design tweak to the block-auction ePBS (the "vanilla" ePBS version that currently has "the scene" in ethereum R&D[[9]](#resources)): the Beacon proposers can commit to execution proposers only, so there would be no commitments to the contents of the execution payload, which solves the timing games issues that the block-auction ePBS version faces. 
+
+**block-auction ePBS vs slot-auction ePBS**
+
+![](/docs/wiki/research/img/ba-ePBS.webp)
+![](/docs/wiki/research/img/sa-ePBS.webp)
+
+The improvement sa-ePBS brings, comes however with the technical cost of dealing with equivocations and head split views, and that [is not a trivial problem to solve](#open-issues).
+
+
+**How do ET solve Timing games?**
+
+Timing games occur when a proposer delays as much as possible the proposing of their block in order to maximize the value it could get for it. In block-auction ePBS, as the execution payloads are determined at the release of the beacon block, the timing games will be played by validators that could attempt to delay the delivery of the beacon block for as much time as possible, in order to commit to the highest bid possible, so it would have the incentives to act rationally. This is what ET proposes, an even further consensus-execution separation, with no commitments from validator-as-proposer to the execution payload:
```

Comment:
modified the paragraph: 
**How do ET contain Timing games?**

Timing games occur when a proposer delays as much as possible the proposing of their block in order to maximize the value it could get for it. In block-auction ePBS, as the execution payloads are determined at the release of the beacon block, the timing games will be played by validators that could attempt to delay the delivery of the beacon block for as much time as possible, in order to commit to the highest bid possible.

ETs proposes, an even further consensus-execution separation to ePBS, with no commitments from validator-as-proposer to the execution payload, thus containing timing games to execution proposing phase, so that the consensus-critical functions are not subject to validators having the incentives to act rationally: